### ISSUE TITLE
Create new workspaces for examples, cli

### ISSUE THREAD

**Issue by decentralgabe** [TBD54566975/ssi-sdk#199]
Since we're upgrading to go 1.19 we'll gain access to workspaces (#198)

It may make sense to create separate modules for examples and the up-coming CLI: https://go.dev/ref/mod#workspaces

@andorsk what do you think?

**Comment by andorsk** [comment]
@decentralgabe good idea. at the face of it sounds good. Will look into this more later, but appreciate you assigning it to me.  Will revert on this issue later.
